### DISCUSSION TITLE
Is there a command to tell if a repo is locked or unlocked

### DISCUSSION THREAD

**Discussion by Cadair** [AGWA/git-crypt#250] [Q&A] [answered]
I would like to know if there is a way I can tell if a repo is locked or not. I would like to integrate it into scripts and my prompt, so I don't forget to unlock as much.

Thanks!

**Comment by alerque** [comment, marked as answer]
There is currently no such command built in, but an open request for one (see #69) also has some suggestions on how you can script this.

**Reply by Cadair** [reply]
How did I manage to miss that issue in my searching. I will put what I ended up scripting over there. Thanks.
